[2024-04-12T18:32:16.764+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T18:32:14.418001+00:00 [queued]>
[2024-04-12T18:32:16.776+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T18:32:14.418001+00:00 [queued]>
[2024-04-12T18:32:16.777+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-04-12T18:32:16.798+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): insert_qdrant> on 2024-04-12 18:32:14.418001+00:00
[2024-04-12T18:32:16.805+0000] {standard_task_runner.py:60} INFO - Started process 476 to run task
[2024-04-12T18:32:16.810+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '12345678', 'insert_qdrant', 'manual__2024-04-12T18:32:14.418001+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpwxnnyv1s']
[2024-04-12T18:32:16.814+0000] {standard_task_runner.py:88} INFO - Job 13: Subtask insert_qdrant
[2024-04-12T18:32:16.887+0000] {task_command.py:423} INFO - Running <TaskInstance: 12345678.insert_qdrant manual__2024-04-12T18:32:14.418001+00:00 [running]> on host 066a79a1baa5
[2024-04-12T18:32:17.042+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='12345678' AIRFLOW_CTX_DAG_ID='12345678' AIRFLOW_CTX_TASK_ID='insert_qdrant' AIRFLOW_CTX_EXECUTION_DATE='2024-04-12T18:32:14.418001+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-12T18:32:14.418001+00:00'
[2024-04-12T18:32:17.139+0000] {python.py:202} INFO - Done. Returned value was: {'status': 'success', 'message': 'Data inserted to Vector DB successfully'}
[2024-04-12T18:32:17.197+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=12345678, task_id=insert_qdrant, execution_date=20240412T183214, start_date=20240412T183216, end_date=20240412T183217
[2024-04-12T18:32:17.241+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-12T18:32:17.279+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
